src/controller/java/CHIPDeviceController-JNI.cpp

```diff
@@ -51,6 +51,9 @@ using namespace chip::DeviceController;
 #define JNI_METHOD(RETURN, METHOD_NAME)                                                                                            \
     extern "C" JNIEXPORT RETURN JNICALL Java_chip_devicecontroller_ChipDeviceController_##METHOD_NAME
 
+#define JNI_ANDROID_CHIP_STACK_METHOD(RETURN, METHOD_NAME)                                                                                            \
```

Comment:
should we move the methods to a separate `AndroidChipStack-JNI.cpp` file? It may be easier to follow if names are consistent between java and cpp source files.

Maybe separate issue/PR since I imagine that will contain larger changes.

Reply:
Sure. Filed #3586 for it.